strapi/design-system
# Migrate JSONInput to TypeScript

### What does it do?

- Migrates the JSONInput component to TypeScript
- Upgrades react-codemirror to latest (I no longer have the bug I had before that forced me to lock the version 🤷‍♂️ )

### Why is it needed?

To gradually migrate the codebase to TypeScript

### How to test it?

- Make sure the input still works as expected in Storybook and Strapi (Content manager and audit logs)
  - For Strapi I recommend creating a build in the DS copying the dist to replace the dist in Strapi (for some reason with a link I get this error: "setTableValues must be initialized via the RawTableContext.Provider" when opening the modal for audit logs)
 - Check the types (I am not a TypeScript expert 🙃 )
